packages/composite-checkout/src/lib/payment-methods/existing-credit-card.js

```diff
@@ -152,131 +144,48 @@ export function createExistingCardMethod( {
 		),
 		submitButton: <ExistingCardPayButton id={ id } />,
 		inactiveContent: (
-			<ExistingCardSummary cardholderName={ cardholderName } brand={ brand } last4={ last4 } />
+			<ExistingCardSummary
+				cardholderName={ cardholderName }
+				cardExpiry={ cardExpiry }
+				brand={ brand }
+				last4={ last4 }
+			/>
 		),
 		getAriaLabel: () => `${ brand } ${ last4 } ${ cardholderName }`,
 	};
 }
 
+function formatDate( cardExpiry ) {
+	const expiryDate = new Date( cardExpiry );
+	const formattedDate = `${ expiryDate.getMonth() + 1 }/${ expiryDate
```

Comment:
This is _probably_ fine, but I think using the built-in date formatting might be more idiomatic and easier to change (also if this is something we want to localize, it will be easier to do that). So perhaps something like this:

```suggestion
	const formattedDate = expiryDate.toLocaleDateString( 'en-US', { month: '2-digit', year: '2-digit' } );
```

The main difference is that for January 20th, your version would print `1/20` and the above version would print `01/20`. If you still want `1/20`, you can change it to:

```suggestion
	const formattedDate = expiryDate.toLocaleDateString( 'en-US', { month: 'numeric', year: '2-digit' } );
```

Reply:
Thanks for this. I'll take your suggestion — it looks a lot cleaner than what I did.